Sources/Extensions/Foundation/UserDefaultsExtensions.swift

```diff
@@ -37,5 +37,33 @@ public extension UserDefaults {
 	public func date(forKey key: String) -> Date? {
 		return object(forKey: key) as? Date
 	}
+    
+    /// SwifterSwift: Retrieves a Codable object from UserDefaults.
+    ///
+    /// - Parameters:
+    ///   - type: Class that conforms to the Codable protocol.
+    ///   - key: key to find the Codable object for.
+    /// - Returns: Codable object for key (if exists).
+    public func codable<T: Codable>(_ type: T.Type, with key: String) -> Codable? {
```

Comment:
I think using some name like `object` would be better? The function signature gives context to the fact that it is codable.

Reply:
Realizing that, I think it makes sense. Will update it accordingly. Also, regarding your general comment (I don't know how to reply to that comment so I'll do it here instead), I've read the Contribution Guidelines but I just overlooked the checkbox for that 😄 . Would you like for me to break down this PR into smaller chunks?

Reply:
@jason-ingenuity If it's not too much trouble can you make it 3 PRs? Defaults, Double, and UICollectionView/UITableView? This makes it easier to: a) Review it b) Accept the PR without rejecting the whole thing from a single flawed area